src/main/java/com/denizenscript/depenizen/bukkit/commands/quests/QuestsCommand.java

```diff
@@ -0,0 +1,221 @@
+                + (questId != null ? questId.debug() : "") 
+                + (stageNum != null ? stageNum.debug() : "") 
+                + (points != null ? points.debug() : "") 
+                + (state != null ? state.debug() : ""));
+
+        switch (Action.valueOf(action.asString().toUpperCase())) {
+
+            case ADD: {
+
+                if (questId != null && state != null) {
+                    for (Quest quest : quests.getQuests()) {
+                        if (quest.getId().equals(questId.asString()) && player != null) {
+                            quests.getQuester(player.getPlayerEntity().getUniqueId()).takeQuest(quest, state.asBoolean());
+                            break;
+                        }
+                    }
+                }
+                else if (points != null && points.asInt() > 0) {
+                    Quester quester = quests.getQuester(player.getPlayerEntity().getUniqueId());
+                    quester.setQuestPoints(quester.getQuestPoints() + points.asInt());
+                    reloadData(quester);
+                }
+                
+                break;
+
+            }
+            case REMOVE: {
+                
+                if (questId != null && state != null) {
+                    for (Quest quest : quests.getQuests()) {
+                        if (quest.getId().equals(questId.asString()) && player != null) {
+                            Player p = player.getPlayerEntity();
+                            Quester quester = quests.getQuester(p.getUniqueId());
+                            quester.hardQuit(quest);
+                            if (state.asBoolean() == true && p.isOnline()) {
+                                p.sendMessage(Lang.get(p, "questQuit").replace("<quest>", quest.getName()));
+                            }
+                            reloadData(quester);
+                            quester.updateJournal();
+                            break;
+                        }
+                    }
+                }
+                else if (points != null && points.asInt() > 0) {
+                    Quester quester = quests.getQuester(player.getPlayerEntity().getUniqueId());
+                    int n = quester.getQuestPoints() - points.asInt();
+                    quester.setQuestPoints(n <= 0 ? 0 : n);
+                    reloadData(quester);
+                }
+                
+                break;
+                
+            }
+            case SET: {
+
+                if (questId != null && stageNum != null && stageNum.asInt() > 0) {
+                    for (Quest quest : quests.getQuests()) {
+                        if (quest.getId().equals(questId.asString()) && player != null) {
+                            try {
+                                quest.setStage(quests.getQuester(player.getPlayerEntity().getUniqueId()), stageNum.asInt());
+                            } catch (InvalidStageException e) {

```

Comment:
Always a newline after `}`

Reply:
Never really liked how that looks, but it shall be done.